go/vt/topo/topoproto/tablet_test.go

```diff
@@ -118,6 +118,7 @@ func TestIsTabletsInList(t *testing.T) {
 	}
 
 	for _, testcase := range testcases {
+		testcase := testcase // capture range variable
```

Comment:
Can you please add a comment here as well just to be sure nobody comes along later and removes this "unnecessary" line :) Something like this:
```
// We create an explicit copy of the range variable for each parallel runner
// to be sure that they each run as expected. You can see more information on
// this here: https://pkg.go.dev/testing#hdr-Subtests_and_Sub_benchmarks
// The problem is covered by govet's rangeloop check:
//   https://github.com/golangci/govet/blob/master/rangeloop.go
```

Otherwise it LGTM. Thank you for contributing to Vitess! ❤️

Reply:
Thanks for reviewing!

I included the comment as you suggested, except for the last two lines, as `vet` does not capture these types of bugs right now (https://github.com/golang/go/issues/16520).

As a side note, the dynamic race detector that ships with Go (`go test -race`) is also not able to detect bugs of this type because of https://github.com/golang/go/issues/35670.